tests/dummy/app/controllers/modules/modal.js

```diff
@@ -1,5 +1,6 @@
 /* global alert */
 import Ember from 'ember';
+import $ from 'jquery';
```

Comment:
I would change the usages of `$...` to `Ember.$...` instead. Or, change this line to `const $ = Ember.$;` But :man_shrugging: